### PR TITLE
IE support for paste content

### LINKED ISSUE #111: Paste issues in IE 9+
When trying to bind a custom paste function, on IE I encounter this problem with this line (ng-wig.js, line ~151).

```
var pasteContent = (event.originalEvent || event).clipboardData.getData('text/plain');
```

Throws an error on IE, due to it not supporting `text/plain` and only `Text`. It pastes the raw data still, but I am unable to fire my custom function.

### PR DESCRIPTION
**Pull request by bampakoa**
Fixes #111 References #112

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -94,7 +94,9 @@ or via bower:
 
     ```<ng-wig ng-model="text1" on-paste="onPaste($event, pasteContent)"></ng-wig>```    
     
-
+###Clear Styles button (plugin) ([plunker](https://plnkr.co/edit/j8FtcMAVkLSztZ6V0ION?p=preview))
```
Comment (stevermeister) [inline README.md]: 👍

File: src/javascript/app/ng-wig/ng-wig.component.js [hunk 2]
```diff
@@ -58,7 +58,13 @@ angular.module('ngWig')
           return;
         }
 
-        let pasteContent = (event.originalEvent || event).clipboardData.getData('text/plain');
+        let pasteContent;
+        if (window.clipboardData && window.clipboardData.getData) { // IE
```
Comment (stevermeister) [inline src/javascript/app/ng-wig/ng-wig.component.js]: are you sure about such check? 

    window.clipboardData


may be we could go with this solution?
http://stackoverflow.com/a/6804718/274500

     if (/text\/html/.test(event.clipboardData.types))

Comment (bampakoa) [inline src/javascript/app/ng-wig/ng-wig.component.js]: Angular UI also had this problem and they fixed it in [this](https://github.com/angular-ui/ui-select/issues/789) way. So, I guess it shall be ok. I have also tested it with [this](https://plnkr.co/edit/J4OIhggQYzJ5VJNn7iLf?p=preview) plunker

Comment (stevermeister) [inline src/javascript/app/ng-wig/ng-wig.component.js]: ok, convinced

**Comment by stevermeister** [comment]
thank you for important fix for IE 👍

**Pull request merged**